LogicMonitor.DataSDK/Api/Logs.cs

```diff
@@ -123,7 +123,13 @@ public override void _mergeRequest()
 
         }
 
-        
-        
+        private string BuildBody(LogsV1 log)
```

Comment:
Why is this method required ? This is not used anywhere.

Reply:
Oops, that isn't even called.. Please remove.